OpenSim/Simulation/Model/Muscle.cpp

```diff
@@ -115,6 +139,10 @@ void Muscle::constructProperties()
     constructProperty_max_contraction_velocity(10.0);
     constructProperty_ignore_tendon_compliance(false);
     constructProperty_ignore_activation_dynamics(false);
+
+    // By default the min and max controls on muscle are 0.0 and 1.0
+    setMinControl(0.0);
+    setMaxControl(1.0);
```

Comment:
This already happens in Muscle's constructor. Either remove the calls from the constructor or from here?